### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

**Comment by Thomasalex2** [comment]
Your E-comm UI looks really good. Didn't find anything to point out in the diff checker, but there seems to be a lot of bottom-padding on your nav bar items

**Review by keyurshah339 — COMMENTED** [review]
clean code ,need a bug fix
on hover of buttons or elements adjacent elements also get affected

File: templates/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+
+    </p>
+  </div>
+  <header>
+    <nav class="nav">
+      <div class="start">
+        <h2 class="theme-heading primary-col">Thrify with me</h2>
+      </div>
+      <ul class="list list-inline f-m">
+        <li class=''>
```
Comment (muskanbagrecha) [inline templates/wishlist.html:24]: Can remove this class

**Comment by AagmanSuri** [comment]
code looks good to merge

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 👍

File: templates/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+            <div class="input-container ">
+              <input placeholder=" " class="input w-full">
```
Comment (muskanbagrecha) [inline templates/wishlist.html:26]: Placeholder is empty

File: templates/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+
+
+<section class="product-container">
+
+  <section class="grid-two-layout-container">
+    <section class="grid-two-layout-left">
+    <div class="container fl-end">
+      <article class="card card-shadow w-s">
+        <div class="background">
+          <img class="product-img-s" src="/media/products/women/dress/1.webp" alt="">
```
Comment (muskanbagrecha) [inline templates/wishlist.html:92]: Add alt

File: templates/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+          <ul class="list list-inline">
+            <li>Instagram</li>
+            <li>Facebook</li>
+            <li>Youtube</li>
+          </ul>
+        </div>
+    </article>
+
+    <article class="grid-three-layout-right landing-footer-section">
+      <div class="">
```
Comment (muskanbagrecha) [inline templates/wishlist.html:317]: Can remove empty class

File: css/styles.css [hunk 5]
```diff
@@ -125,3 +125,12 @@ a {
   left: 35vw;
   z-index:10;
 }
+
+.out-of-stock{
+  opacity:0.4;
+}
+
+.out-of-stock-head{
+  position:relative;
```
Comment (muskanbagrecha) [inline css/styles.css:134]: This might not be needed. Try removing it and check.

**Review by muskanbagrecha — APPROVED** [review]
UI:
1. Increase letter spacing in button.
2. Increase font weight of sale label (maybe even price)
3. Add hover on cart button

File: templates/wishlist.html (hunk 2 above)
Comment (keyurshah339) [inline templates/wishlist.html:26]: no need for empty placeholder

File: templates/wishlist.html [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+              <p class="original-price">1599 Rs</p>
+              <p class="sale-price">780 Rs</p>
+            </div>
+
+        </div>
+        </article>
+
+        <article class="card card-shadow w-s out-of-stock">
+          <div class="background">
+            <img class="product-img-s" src="/media/products/women/dress/2.webp" alt="">
```
Comment (keyurshah339) [inline templates/wishlist.html:115]: add alt

**Pull request merged**